testGradient.m

```diff
@@ -140,21 +157,20 @@
     eval(['l = struct.' il ';']);
 end
 
-for i = 1:length(theta)
-    i
+for i = 1:length(plist)

```

Comment:
Could we please avoid the counter i since it'c acutally the imaginary unit in Matlab? iPar would be better here...

Reply:
done